site/public/js/polls.js

```diff
@@ -109,3 +109,13 @@ function updateDropdownStates(curr_state, cookie_key) {
 function importPolls() {
     $('#import-polls-form').submit();
 }
+
+function toggleTimerInputs() {
+    const enableTimerCheckbox = $('#enable-timer');
+    if (enableTimerCheckbox.prop('checked')) {
```

Comment:
This could be inlined since it's only used once.  Note: this is just a suggestion, so it is not a blocker.

Reply:
Makes sense, added suggestion